publish.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
Are you planning on making this executable? If so, you're missing a couple of things:

* need the `#!/usr/bin/env node` preamble at the top of this
* Add it to the `bin` key in the `package.json`, so it's like:

```
{
  "name": "screwdriver-template-main",
  ...
  "bin": "./package.js"
}
```

Even better, you could specify the binary name with: 

```
{
  "name": "screwdriver-template-main",
  ...
  "bin": {
    "template-publisher": "./package.js"
  }
}
```

Doc for reference: https://docs.npmjs.com/files/package.json#bin